app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class UserMailer < ApplicationMailer
+  def send_confirmation_email(user)
+    @user = user
+    mail(to: @user.email, subject: 'Welcome to WellCare - Confirm Your Account')
```

Comment:
Suggestion: I think converting all hardcoded string to i18n will be a better practice

Reply:
@asis-wtag Thanks for the suggestion bhai, I will introduce `i18n` in a later PR.